doc/DoxyFile.NATS.Client.in

```diff
@@ -2279,7 +2279,8 @@ INCLUDE_FILE_PATTERNS  =
 # This tag requires that the tag ENABLE_PREPROCESSING is set to YES.
 
 PREDEFINED             = BUILD_IN_DOXYGEN \
-                         @NATS_DOC_INCLUDE_STREAMING@
+                         @NATS_DOC_INCLUDE_STREAMING@ \
+                         @NATS_DOC_INCLUDE_TLS@
```

Comment:
You need to do a `make edit_cache` and enable the `NATS_UPDATE_DOC` option. After that, do a `make` and it should rebuild but will generate the `DoxyFile.NATS.Client` (notice the extension that is not `.in`), which will be added to the changes.